node_scenarios/node_start_stop_scenario.yml

```diff
@@ -0,0 +1,3 @@
+label_selector:                                              # This enables the nodes with the specified label selector to be selected for chaos injection
+instance_kill_count:                                         # Number of times to inject the scenario in the nodes
+timeout:                                                     # Set the time out after which the test can fail in case the node is not in the expected state after chaos scenario injection
```

Comment:
Think the config should accept a list of node scenarios ( start, stop, restart, terminate ) similar to how powerfulseal does it - https://github.com/bloomberg/powerfulseal/blob/master/tests/policy/example_config2.yml except that we will ask user for list of actions ( start, stop, restart, terminate ) instead of ip address's/hostnames. Thoughts?